core/gather/driver/wait-for-condition.js

```diff
@@ -260,7 +261,7 @@ function waitForCPUIdle(session, waitForCPUQuiet) {
   const executionContext = new ExecutionContext(session);
   /** @type {Promise<void>} */
   const promise = new Promise((resolve, reject) => {
-    executionContext.evaluate(registerPerformanceObserverInPage, {args: []})
+    executionContext.evaluate(registerPerformanceObserverInPage, {args: [], useIsolation: true})
```

Comment:
Interesting! I don't have any intuition for PerformanceObserver behavior in an isolated context. What happens?

Reply:
From my testing, it works the same. We still get the longtask events.